packages/create-catalyst/src/utils/clone-catalyst.ts

```diff
@@ -7,12 +7,14 @@ export const cloneCatalyst = ({
   projectName,
   projectDir,
   ghRef,
+  repository,
 }: {
   projectName: string;
   projectDir: string;
-  ghRef?: string;
+  ghRef: string | undefined;
+  repository: string | undefined;
```

Comment:
Not entirely confident that this is best practice, I just thought it would be more helpful to explicitly require all params be passed to the function, even if the values for some params is `undefined`.